db/migrate/20201214192048_add_enabled_to_external_project.rb

```diff
@@ -0,0 +1,5 @@
+class AddEnabledToExternalProject < ActiveRecord::Migration[6.0]
+  def change
+    add_column :external_projects, :enabled, :boolean, default: true
```

Comment:
always use `null: false` for boolean columns https://thoughtbot.com/blog/avoid-the-threestate-boolean-problem